ddtrace/contrib/aiopg/__init__.py

```diff
@@ -1,7 +1,5 @@
-"""Instrument aiopg to report Postgres queries.
-
-``patch`` will automatically patch your aiopg connection to make it work.
-::
+"""Instrument aiopg to report Postgres queries. ``patch_all()`` will automatically
```

Comment:
I think  it was copied from other integrations (such as postgres). At some point I want to revamp our docs because in general they're a bit confusing for what is auto instrumented and what is not.

Here I can write: "Instrument ``aiopg`` to report a span for each executed Postgres queries." maybe?